cmake/platform_linux_desktop.cmake

```diff
@@ -99,6 +99,8 @@ function(setup_litecore_build)
            ${liteCoreVariant} INTERFACE
            Threads::Threads
         )
+        target_link_libraries(${liteCoreVariant} PRIVATE ${ZLIB_LIB})
```

Comment:
I don't know why, after refactoring, the linker cannot find functions in zlib. Prior to this PR, I don't see where ZLIB_LIB is used.

Reply:
It is used in platform_linux.cmake in this case.  Since you moved the definition out of its current location it is no longer defined when it reaches there.

Reply:
I see. Will do it.